### PR TITLE
Issue #431 adding passivating scope to a non passivation capable bean…

### PR DESCRIPTION
**Pull request by jeanouii**
… must result in a deployment problem (DeploymentException subclass)

### CODE REVIEW

**Review by manovotn — APPROVED** [review]
Thanks for the PR.
I am fine with the change but will wait for second pair of eyes before merging the change for this challenge.

**Review by Ladicek — APPROVED** [review]

**Review by manovotn — APPROVED** [review]

**Pull request merged**
